### PR TITLE
refactor!: update to cheminfo-types

### PR DESCRIPTION
**Pull request by maasencioh**
BREAKING CHANGE: rename data to variables

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,14 +45,22 @@
...
-    "ts-jest": "~26.4.4",
-    "typescript": "~4.1.2"
+    "@types/jest": "^27.0.2",
+    "@typescript-eslint/eslint-plugin": "^5.0.0",
+    "@typescript-eslint/parser": "^5.0.0",
+    "eslint": "^8.1.0",
+    "eslint-config-cheminfo-typescript": "^10.2.2",
+    "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-jest": "^25.2.1",
+    "eslint-plugin-jsdoc": "^37.0.0",
```
Comment (targos) [inline package.json:59]: this shouldn't be necessary

File: package.json [hunk 2]
```diff
@@ -44,14 +45,22 @@
...
-    "eslint": "~7.14.0",
-    "eslint-config-cheminfo-typescript": "~8.0.5",
-    "jest": "~26.6.3",
-    "prettier": "~2.2.0",
-    "rimraf": "~3.0.2",
-    "ts-jest": "~26.4.4",
-    "typescript": "~4.1.2"
+    "@types/jest": "^27.0.2",
+    "@typescript-eslint/eslint-plugin": "^5.0.0",
+    "@typescript-eslint/parser": "^5.0.0",
```
Comment (targos) [inline package.json:54]: same

Comment (maasencioh) [reply package.json:54]: I have this error if I delete it
```
ESLint: 8.1.0

Error: Failed to load parser '@typescript-eslint/parser' declared in '.eslintrc.yml » eslint-config-cheminfo-typescript#overrides[0]': Cannot find module '@typescript-eslint/parser'
Require stack:
- /home/runner/work/ndim-parser/ndim-parser/node_modules/eslint-config-cheminfo-typescript/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.resolve (internal/modules/cjs/helpers.js:99:19)
    at Object.resolve (/home/runner/work/ndim-parser/ndim-parser/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:1088:46)
    at ConfigArrayFactory._loadParser (/home/runner/work/ndim-parser/ndim-parser/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3235:39)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/home/runner/work/ndim-parser/ndim-parser/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3030:43)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigData (/home/runner/work/ndim-parser/ndim-parser/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:2971:20)
    at _normalizeObjectConfigData.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/home/runner/work/ndim-parser/ndim-parser/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3064:25)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
```

Comment (targos) [reply package.json:54]: you need npm 7 or 8 (node 16 or 17)

**Pull request merged**